### ISSUE TITLE
Select Size before pasting Image from Clipboard

### ISSUE THREAD

**Issue by jdujava** [xournalpp/xournalpp#5384]
### Is Your Feature Request Related to a Problem?

Image pasted from the clipboard is often overly big and I need to resize it manually (essentially the same as #4183, but specifically for clipboard pasting).

### Feature Description

Feature introduced by #4966 is very useful, it would be amazing if the size chooser would be automatically triggered when pasting an image from the clipboard (this behaviour could be opt-in, or possibly on a different binding such as `Ctrl+Shift+v`).

@p0mm, what do you think about it?

**Comment by p0mm** [comment]
This sounds pretty cool!
 I am certainly a fan of pictures not being a 'random' size when inserting :)

Just to make sure i am understanding your suggestion, what the automatic triggering of size chooser looks like:
- you'd choose to have this behavior (in the preference settings) when pasting from clipboard
- or use a special binding when inserting the image from the clipboard, `Ctrl+Shift+v` instead of `Ctrl+v` 
- you would then not see the image, but instead get the little cross cursor and be able to choose the size by dragging (this for sure should be able to use the already existing code!)
- after that the image appears in your chosen size (if you chose zero space, either noting happens or image gets added in original size?)

This should be fairly easy to implement - if no one starts working on this till the holidays I'd be happy to implement this, currently I am preoccupied with studying :<    
Thank you so much for pinging me in this btw!

**Comment by jdujava** [comment]
Yes, that's exactly what I had in mind. I'm not sure what specifics would be the best, so I will just throw out some ideas:
- to have: _preference setting_, or _special binding_ (`Ctrl+Shift+v`), or maybe _both_
- I think `Esc` should cancel the action of pasting
- on choosing zero space: _nothing happens_ (cancels the action) or inserts with _original size_
- something like double pressing `Ctrl+v` could insert the image with original size (in the absence of a dedicated binding, seems quicker then `Ctrl+v` and extra zero space tap to insert original size) 
- the copy of an image/object already from Xournalpp should probably retain its size (in the absence of a dedicated binding)

Seems to me that adding a dedicated binding would be really versatile. However, as of now `Ctrl+Shift+v` is mapped to `Vertical Space`, so that should be somehow sorted out (`Ctrl+Shift+s` is free and kinda makes sence for `Space`, maybe?).

> This should be fairly easy to implement - if no one starts working on this till the holidays I'd be happy to implement this, currently I am preoccupied with studying :<

Sounds great, looking forward!
